tutorial/interpreter.po

```diff
@@ -221,7 +234,7 @@ msgstr ""
 msgid ""
 "For example, to declare that Windows-1252 encoding is to be used, the first "
 "line of your source code file should be::"
-msgstr ""
+msgstr "例如，要聲明使用Windows-1252的編碼，您的原始碼首行應為::"
```

Comment:
As described in the third point in the translation guidelines [1], "Windows-1252" should have a space both before and after the word.

The colon, again, should follow the rST guidelines.

[1] https://github.com/python/python-docs-zh-tw#id7